internal/docker/docker.go

```diff
@@ -90,3 +118,26 @@ func ConnectToNetwork(containerID, network string) error {
 	}
 	return nil
 }
+
+// InspectContainers function inspects selected Docker containers.
+func InspectContainers(containerIDs ...string) ([]ContainerDescription, error) {
+	args := []string{"inspect"}
+	args = append(args, containerIDs...)
+	cmd := exec.Command("docker", args...)
```

Comment:
We could use the [docker API](https://pkg.go.dev/github.com/docker/docker/client#Client.ContainerInspect) directly, so we don't have to define and parse the docker API structs here. Or you are explicitly avoiding adding this new dependency?

Reply:
I think we decided to go with "docker" as it was quicker. We didn't have to struggle with different Docker APIs (different versions) and it's consistent with "docker-compose". I'm fine with creating an issue to replace all "docker" calls with a library and APIs.